app/models/common/MembersSummary.scala

```diff
@@ -51,8 +53,13 @@ object MembersSummary {
       }
     )
 
-  def readsMember(implicit messages: Messages): Reads[MembersSummary] = {
-    JsPath.read(readsMemberSummary)
+  def readsMember(eventType: EventType)(implicit messages: Messages): Reads[MembersSummary] = {
+    JsPath.read(readsMemberSummary(eventType))
+  }
+
+  def memberValuePath(eventType: EventType): JsPath = eventType match {
+    case Event6 => JsPath \ "AmountCrystallisedAndDate" \ "amountCrystallised"
```

Comment:
Use lowercase for "amountCrystallised..."

Reply:
This just matches what's in the JSON for UserAnswers and wasn't a naming decision made on this ticket

Reply:
Ah OK. Well it's not consistent but for now I guess we can leave it